src/components/Masthead.vue

```diff
@@ -3,7 +3,8 @@
     <div class="wrapper">
 
         <header>
-            <div class="logo" v-bind:class="{ login : $route.path==='/login' || $route.path==='/user' || $route.path==='/request' || $route.path==='/change-password' }">
+        
+            <div class="logo" v-bind:class="{ login : $route.path==='/login' || $route.path==='/user' || $route.path==='/request' || $route.path==='/change-password' || $route.path==='/privacy-policy' }">
```

Comment:
Are all of these pages which don't require auth to view? Why not just check the meta for the route.